test_common/harness/imageHelpers.h

```diff
@@ -495,6 +495,7 @@ void read_image_pixel(void *imageData, image_descriptor *imageInfo, int x,
     }
     else if (format->image_channel_order == CL_LUMINANCE)
     {
+        outData[0] = tempData[0];
```

Comment:
Do we need the same change for setting `outData[0]` in the  `CL_INTENSITY` case too? Haven't worked with images much, so might be reading things wrong.

Reply:
Yup, well spotted, it was broken too. I've updated the PR to fix that one too.